client/signup/steps/about/index.jsx

```diff
@@ -281,26 +281,22 @@ class AboutStep extends Component {
 		const userExperienceInput = this.state.userExperience;
 		const siteTopicInput = formState.getFieldValue( this.state.form, 'siteTopic' );
 
+		const eventAttributes = {};
+
 		//Site Title
 		if ( siteTitleInput !== '' ) {
 			siteTitleValue = siteTitleInput;
 			this.props.setSiteTitle( siteTitleValue );
 			this.props.setDomainSearchPrefill( siteTitleValue );
 		}
 
-		this.props.recordTracksEvent( 'calypso_signup_actions_user_input', {
-			field: 'Site title',
-			value: siteTitleInput || 'N/A',
-		} );
+		eventAttributes.site_title = siteTitleInput || 'N/A';
```

Comment:
Would it break anything to just send `null`? I realize you aren't here to change that, but what if someone's title is "N/A"? 🤔

Reply:
yeah - good point - i'll deploy this now but perhaps change this later - what do you think @fditrapani?